webcrypto/aes.go

```diff
@@ -395,16 +395,16 @@ func (acp *AesCtrParams) Decrypt(ciphertext []byte, key CryptoKey) ([]byte, erro
 	return plaintext, nil
 }
 
-// Ensure that AesCtrParams implements the EncryptDecrypter interface.
-var _ EncryptDecrypter = &AesCtrParams{}
+// Ensure that AESCtrParams implements the EncryptDecrypter interface.
+var _ EncryptDecrypter = &AESCtrParams{}
 
-// AesGcmParams represents the object that should be passed as the algorithm parameter
+// AESGcmParams represents the object that should be passed as the algorithm parameter
 // into `SubtleCrypto.Encrypt`, `SubtleCrypto.Decrypt`, `SubtleCrypto.WrapKey`, or
 // `SubtleCrypto.UnwrapKey`, when using the AES-GCM algorithm.
 // As defined in the [specification].
 //
 // [specification]: https://www.w3.org/TR/WebCryptoAPI/#aes-gcm-params
-type AesGcmParams struct {
+type AESGcmParams struct {
```

Comment:
```suggestion
type AESGCMParams struct {
```
It is an acronym so it should be all uppercase. I can see the readability issue. I'm not against the current version, I see this suggestion has a readability issue. `AES_GCMParams` should also works.

What is your idea here?

Reply:
You're right, GCM stands for Galois Counter Mode, and is an acronym, we should probably uppercase it indeed, I just didn't pay enough attention ✋🏻